docs/commands.rst

```diff
@@ -44,3 +45,177 @@ BUNDLE
 ^^^^^^
 
 The BUNDLE command gathers files necessary to create a prediction package from the output of the previous commands. A prediction package contains a model file plus associated configuration data, and can be used to make predictions on new imagery in a deployed application.
+
+.. _aux commands:
+
+Auxiliary (Aux) Commands
+------------------------
+
+Raster Vision utilizes *auxiliary commands* for things like data preperation. These are commands that do not run in the normal ML pipeline (e.g., if one were to run run ``rastervision run`` without an command specified). Auxiliary commands normally do not have the same type of implicit configuration setting as normal commands; because of this, file paths are often set explicity, and these commands are often configured and returned from an ``ExperimentSet`` method directly, instead of implicity created through the ``ExperimentConfig``.
+
+Configuring Aux Commands
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are two ways to configure an Aux command: one is through custom configuration set on an ``ExperimentConfig``, and the other is to directly return a ``CommandConfig`` instance from an experiment method. Normally Aux Commands are run separately from the normal experiment workflow, so we suggest returning command configurations as a default.
+
```

Comment:
If you return a command config from an experiment method, does this mean you cannot run any other commands for the experiment defined by that method? Is the reason you added support for the other option (setting the custom config on the `ExperimentConfig`) that you might want to run the aux command plus built-in command as part of a single Batch DAG?